Catalog/Projects/Tax-Calculator/catalog.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
@hdoupe, what is your thinking of the pros/cons of including this file here rather than in Tax-Calculator?

Reply:
I think it should go in Tax-Calculator. That is, each upstream project should have a catalog.json file in their own repo. This should make it easier for the upstream project to update their catalog data. They only have to update data in their own repo. There's no need for them to open a PR here to update their data.

This means that there should be a way for them to easily test that their catalog.json file meets formatting requirements and that the data that it points to meets formatting requirements. There should also be documentation on what the formatting criteria are and how to run the tests. I'll work on this.

Also, we should have a registry of some sort that has the project's name and a link to their catalog.json file so that the builder script can find it.

I guess the downsides are that the upstream project has to keep an extra file in its repo and is responsible for updating it themselves. However, I think it would be better for the upstream projects to have more responsibility in updating their catalog data. They should have a better idea of what information they want to provide and how they want to provide it.

@MattHJensen is this inline with what you are thinking?

Reply:
Yep, that's in line with my thinking.